src/processes.c

```diff
@@ -1529,8 +1529,14 @@ static char *ps_get_owner(pid_t pid)
         uid = atoi (line + 5);
         getpwuid_r (uid, &passwd, passwd_buffer, sizeof(passwd_buffer),
                 &passwd_result);
-        if (passwd_result)
+        if (passwd_result) {
             result = sstrdup (passwd_result->pw_name);
+        } else {
+	    // Send the numeric uid if name is not available.
+            char uid_str[6];  // 5 digits plus string terminator.
+            snprintf(uid_str, sizeof(uid_str), "%u", uid);
```

Comment:
Used uid_t as the type and took your suggestion of using strtoul.

Re: string --> integer --> string: extracting the uid from /proc/<pid>/status returns a string with multiple numbers in it. The atoi/strtoul grabs the first one. Otherwise would have avoided what looks to be a redundant conversion.